modules/alignment.nf

```diff
@@ -225,7 +225,7 @@ process diamond {
         --query-cover ${params.dmnd_min_coverage} \
         --id ${params.dmnd_min_identity} \
         --top ${params.dmnd_top_pct} \
-        --block-size ${task.memory.toMega() / (1024 * 6 * task.attempt)} \
+        --block-size ${task.memory.toMega() / (1024 * 6 )} \
```

Comment:
I'm worried that using `task.memory` will cause this process to not `-resume` if the user increases the memory allowed for this tag.

Reply:
My understand is that this will just let the software know the intended cpus / memory of the task. It's independent of the attempt and thus may work better.